### ISSUE TITLE
Move LogTo to testing section

### ISSUE THREAD

**Issue by ChrisJefferson** [alex-konovalov/gap-lesson#14]
Move LogTo to later on, maybe in testing section.

**Comment by alex-konovalov** [comment]
I agree with this. It will be useful to mention is there as a possible way to create test files, and remove this block from the initial session which is already quite densely packed.

**Comment by alex-konovalov** [comment]
@ChrisJefferson (CC @markuspf) `LogTo` is still there in the beginning of the lesson. Do you remember why your intention was to move it to a later episode?

**Comment by ChrisJefferson** [comment]
I think just a case of, does it have to be introduced so early, when people are just trying to get started. Minimising the number of things they have to do early on would be useful I think.

**Comment by alex-konovalov** [comment]
Still could be useful, as @markuspf pointed out yesterday, for learners to record what they are doing from the beginning.

Difficulties may happen when learners will try to find that file, especially if they are beginners, and this is not a part of a full-scale Software Carpentry workshop, and they have a mix of operating system with different current directories when GAP starts (on Windows, will likely be `C:\Users\Username\`). 

I have tried to address that by calling `ChangeDirectoryCurrent` in "Getting Ready" (http://alex-konovalov.github.io/gap-lesson/) to ensure that they are in the same directory before they start, but then you get learners with IO not compiled by some reason, and again that prevents from a smooth start.

My current suggestion is: keep introducing `LogTo` in the first session with GAP (like the official GAP Tutorial does), but extend the Instructor Guide (http://alex-konovalov.github.io/gap-lesson/guide/) with some additional advice.
